Source/controls/game_controls.cpp

```diff
@@ -15,16 +15,15 @@
 #include "doom.h"
 #include "gmenu.h"
 #include "options.h"
-#include "plrctrls.h"
 #include "qol/stash.h"
 #include "stores.h"
 
 namespace devilution {
 
 bool start_modifier_active = false;
 bool select_modifier_active = false;
-const ControllerButton ControllerButton_ATTACK = ControllerButton_BUTTON_B;
-const ControllerButton ControllerButton_CAST_SPELL = ControllerButton_BUTTON_X;
+const ControllerButton ControllerButtonPrimery = ControllerButton_BUTTON_B;
+const ControllerButton ControllerButtonSecoundery = ControllerButton_BUTTON_X;

```

Comment:
It was a pain to add a review comment for every usage. It's probably better to just use an IDE to rename anyway. 😓 
```suggestion
const ControllerButton ControllerButtonPrimary = ControllerButton_BUTTON_B;
const ControllerButton ControllerButtonSecondary = ControllerButton_BUTTON_X;
```

Reply:
Yeah, don't do that I know how to search and replace :D